kaldb/src/main/proto/manager_api.proto

```diff
@@ -31,6 +31,8 @@ message CreateDatasetMetadataRequest {
   string name = 1;
   // Owner information, maybe be any string
   string owner = 2;
+  // The service name(s) that the dataset is indexing
```

Comment:
This isn't a repeated string, so only one service name can be provided here. If this is supposed to be a regex/glob that could match multiple services, this probably should be named service_name_pattern.